docker-compose.yml

```diff
@@ -5,31 +5,35 @@ services:
     restart: always
     build: ./
     ports:
-      - "8081:80"
+      - "8000:8080"
     environment:
-      # - HYDRUS_SERVER_URL=http://localhost:8080/ ## For linux local
-      - HYDRUS_SERVER_URL=http://192.168.99.100:8081/ ## For windows 10
+      # - HYDRUS_SERVER_URL=http://192.168.99.100:8081/ ## For windows 10
       # - HYDRUS_SERVER_URL=http://54.169.232.177:8080/
+      - DEBUG=1
       - API_NAME=api
-      - DB_URL=postgresql://hydra:hailhydra@postgres_db:5432/hydrus
+      # relative path to the ApiDoc, from the project root
+      - APIDOC_REL_PATH=examples/drones/doc.py
+      - DB_URL=sqlite:///database.db
+      #- DB_URL=postgresql://hydra:hailhydra@postgres_db:9001/hydrus
+      - PORT=8080  # shall match with "ports" above
     networks:
       main:
         aliases:
           - hydrus_server
 
-  postgres:
-    restart: always
-    environment:
-      - POSTGRES_USER=hydra
-      - POSTGRES_PASSWORD=hailhydra
-      - POSTGRES_DB=hydrus
-    image: postgres:latest
-    ports:
-      - "9000:5432"
-    networks:
-      main:
-        aliases:
-          - postgres_db
+#  postgres:
+#    restart: always
+#    environment:
+#      - POSTGRES_USER=hydra
+#      - POSTGRES_PASSWORD=hailhydra
+#      - POSTGRES_DB=hydrus
+#    image: postgres:latest

```

Comment:
We don't need this anymore.

Reply:
we will add that soon. the problem is that the tables creation script is unsafe.